main.py

```diff
@@ -1,4 +1,17 @@
 print('Sup world!')
 
-for _ in range 10:
-  print(f"Go nuts {_} time(s)")
+import socket
+
+class Bot:
+    def __init__(self):
+        self.server = "irc.provisionweb.org"
+        self.port = 6667
+        self.nick = "ShitCodedBot"
+        self.sock = socket.socket()

```

Comment:
Ident should't be required?

Reply:
Well, yeah eventually but we haven't made anything beyond the connect. We can do that now

Reply:
I thought it's required to even connect to irc server